src/main/java/duke/Event.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+class Event extends Task {
+
+    protected String at;
+
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
+    }
```

Comment:
Same thing with the deadline task

```suggestion
    public Event(String description, String eventDate) {
        super(description);
        this.eventDate= eventDate;
    }
```